Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/src/.DS_Store and wxnew/src/.DS_Store differ
Binary files wxorig/src/html/.DS_Store and wxnew/src/html/.DS_Store differ
diff -uNr wxorig/src/html/m_fonts.cpp wxnew/src/html/m_fonts.cpp
--- wxorig/src/html/m_fonts.cpp	2008-01-30 08:31:58.000000000 -0500
+++ wxnew/src/html/m_fonts.cpp	2010-12-29 17:00:58.000000000 -0500
@@ -49,7 +49,7 @@
             }
         }
 
-        if (tag.HasParam(wxT("SIZE")))
+        if (tag.HasParam(wxT("SIZE")) && tag.GetParam(wxT("SIZE")).Length() > 0)
         {
             int tmp = 0;
             wxChar c = tag.GetParam(wxT("SIZE")).GetChar(0);
